README.md

```diff
@@ -10,6 +10,9 @@ The current version of the proxy utilizes Google Cloud Pub/Sub as the backend Pu
 If you're planning on running multiple instances of the proxy for scalability,
 you must avoid running multiple instances on the same host.
 
+If you will be using Google Cloud Pub/Sub, you must set the GCLOUD_PUBSUB_PROJECT_ID environment
```

Comment:
In gcloud (Veneer) libraries we decided to use GOOGLE_CLOUD_PROJECT.
see: https://github.com/GoogleCloudPlatform/gcloud-meta/issues/17